Project.toml

```diff
@@ -11,5 +11,4 @@ Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
 
 [compat]
 julia="^1.5"
```

Comment:
If you're going to drop the julia 1.6 tests, i guess you should also bump the required julia to 1.7 or later?

Otherwise we should probably be testing on the versions we want to support, and the usual way people do that is they test on all three of (oldest supported version, current official release, julia master)

Reply:
not honestly sure why 1.6 didn't work… it said a manifest (this package? a dependency?) was generated with a different version… wasn't sure how to fix this

Reply:
https://github.com/RelationalAI-oss/ServerMetrics.jl/runs/7359426081?check_suite_focus=true